pkg/azuredx/datasource.go

```diff
@@ -37,57 +37,62 @@ func NewDatasource(settings backend.DataSourceInstanceSettings) (instancemgmt.In
 	if err != nil {
 		return nil, err
 	}
+	// TODO(pascaldekloe): We need a way to set oauthPassThru in the plugin instead.
```

Comment:
still relevant?

Reply:
The plugin will not be able to change json data config at this part of the life cycle, so you can remove the TODO. Even though this is not ideal, it's now clearly stated in the docs how to configure OBO using provisioning.